problems/pancake/pancake_problem.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Tuple
+from base.problem import Problem
+from problems.pancake.pancake_state import PancakeState
+from problems.pancake.pancake_action import PancakeAction
+from PIL import Image, ImageDraw
+
+
+class PancakeProblem(Problem[PancakeState, PancakeAction]):
+    def __init__(self, initial: PancakeState):
+        super().__init__(initial)
+        self.n_pancakes = len(initial.pancakes) - 1
+        self.goal = PancakeState(pancakes=[i for i in range(1, self.n_pancakes+2)])
```

Comment:
W znalezionych przeze mnie źródłach naleśniki były podawane jako np (1 3 4 5 2 6) tak że zawsze ostatni numer to był największy numer (talerza). Przez to liczba naleśników jest o jeden mniejsza niż długość listy.

Reply:
Jak dla mnie może być (chociaż nie do końca czuję, po co nam informacja o numerze talerza, skoro zawsze jest największy i zawsze jest na dole). W takim wypadku dodałbym komentarz, który by to "dokumentował".